InvenTree/label/models.py

```diff
@@ -146,6 +146,37 @@ def __str__(self):
         max_length=100,
     )
 
+    multipage = models.BooleanField(
+        default=False,
+        verbose_name=_('Print multiple labels to a single page'),
+    )
+
+    page_width = models.FloatField(
+        default=210,
+        verbose_name=_('Width of one page [mm]'),
+        help_text=_('The number of label columns per page will be automatically calculated from the label\'s width'),
```

Comment:
Using `\'` can make translation strings difficult - please enclose the entire string in double quotes to avoid the slash